tests/bluetooth/bsim/nrf_auraconfig/tests/usecase_defines.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Indexes
+BIG_0=0
+BIG_1=1
+SUB_G_0=0
+SUB_G_1=1
+SUB_G_2=2
+BIS_0=0
+BIS_1=1
+BIS_2=2
+
+# Boolean
+TRUE=1
+FALSE=0
+
+# PHY
+PHY_1M=1
+PHY_2M=2
+PHY_CODED=4
+
+# Locations
+MA=0x00000000
```

Comment:
Consider prepending LOC_

Reply:
Discussed and we want to keep it as close to zephyr defines as possible